ReactNativeClient/package-lock.json

```diff
@@ -4613,7 +4613,8 @@
         },
```

Comment:
Please remove this file from the commit.

Reply:
Done, sorry I didn't take note of this before